packages/@aws-cdk/toolkit/lib/toolkit/toolkit.ts

```diff
@@ -425,7 +425,7 @@ export class Toolkit extends CloudAssemblySourceBuilder implements AsyncDisposab
 
         await ioHost.notify(info('Stack ARN:'));
 
-        await ioHost.notify(data(deployResult.stackArn));
+        await ioHost.notify(info(deployResult.stackArn));
```

Comment:
`data` doesn't exist and i assume that `info` is the better thing to put here anyway

Reply:
I think I marked `data` was deprecated, so this seems alright.